Sasquatch/Sasquatch/Base.lproj/Main.storyboard

```diff
@@ -839,68 +839,96 @@
                                             <rect key="frame" x="0.0" y="0.0" width="375" height="43.5"/>
                                             <autoresizingMask key="autoresizingMask"/>
                                             <subviews>
-                                                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Identity Enabled" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="pAY-e1-ggT">
-                                                    <rect key="frame" x="16" y="11.5" width="123.5" height="20.5"/>
+                                                <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="lTr-Ec-UOq">
+                                                    <rect key="frame" x="0.0" y="0.0" width="375" height="43.5"/>
+                                                    <state key="normal" title="Login"/>
+                                                    <connections>
+                                                        <action selector="identityLogin:" destination="9Ck-4D-Ozb" eventType="touchUpInside" id="X5b-TR-Uhv"/>
+                                                        <action selector="login" destination="utj-Ji-UwU" eventType="touchUpInside" id="YpM-bE-QnO"/>
+                                                    </connections>
+                                                </button>
+                                            </subviews>
+                                            <constraints>
+                                                <constraint firstItem="lTr-Ec-UOq" firstAttribute="leading" secondItem="rfG-En-bfa" secondAttribute="leading" id="KCI-Jt-VXk"/>
+                                                <constraint firstAttribute="bottom" secondItem="lTr-Ec-UOq" secondAttribute="bottom" id="Kgb-sW-JIZ"/>
+                                                <constraint firstAttribute="trailing" secondItem="lTr-Ec-UOq" secondAttribute="trailing" id="SWR-3Y-XvX"/>
+                                                <constraint firstItem="lTr-Ec-UOq" firstAttribute="top" secondItem="rfG-En-bfa" secondAttribute="top" id="gqh-Nd-jns"/>
+                                            </constraints>
+                                        </tableViewCellContentView>
+                                    </tableViewCell>
+                                </cells>
+                            </tableViewSection>
+                            <tableViewSection headerTitle="Miscellaneous" id="3h3-3G-53T">
```

Comment:
Push switch is in Miscellaneous section. Since we have a separate section for Identity, it might be good to add a Push section and move the switch under the section.
Or, we probably consider to have a new tab for Identity. I'd prefer this one but not sure how the UI will look like when we have 6 tabs.
@clpolet @ElektrojungeMS @achocron What do you guys think?

Reply:
+1 for having an Identity tab for usability despite of the ugliness of UI.

Reply:
We don't have enough space for many tabs. Actually, I like the approach from our .NET demo app: Distribute and Push don't have many settings and combined into one tab "Other".

Anyway, I believe this is not a blocker for this PR and we can redesign/refactor it later.

Reply:
Let's do this separately.